modules/gitops-release-notes-1-7-1.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-7-1_{context}"]
+= Release notes for {gitops-title} 1.7.1
+
+{gitops-title} 1.7.1 is now available on {product-title} 4.8, 4.9, 4.10, 4.11, and 4.12.
+
+[id="errata-updates-1-7-1_{context}"]
+== Errata updates
+
+=== RHSA-2023:0467 - {gitops-title} 1.7.1 security update advisory 
+
+Issued: 2023-01-25
+
+{gitops-title} release 1.7.1, which includes security updates, is now available. The list of security fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:0467[RHSA-2023:0467] advisory. 
```

Comment:
Looking at the OpenShift version of these, it seems like this is a standard block of copy. Do we have any leeway to adjust it, or do we need to keep it exactly like this? 

If we're able to change it each time we add this Errata block - what do you think about removing the first sentence? Given that we're putting the Errata in with each version's RNs it feels repetitive to say that the release is now available. We could add the "which includes security updates" part to Line 9 if we want it to be mentioned somewhere?

Reply:
@Preeticp, need your inputs here. 
The first sentence suggests that a newer version (1.7.1) is available, though the same information is mentioned a  few lines above as well. So, this makes the content a bit redundant. I guess, we can remove the first sentence and start the para from the second sentence `The list of security fixes that are included in the update is documented...`  If you look at the preview, you can guage it better: https://55464--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes.html#gitops-release-notes-1-7-1_gitops-release-notes

Reply:
@shipsing +1, tiny tweak: The list of security fixes that are included in this release is documented in the [RHSA-2023:0467](https://access.redhat.com/errata/RHSA-2023:0467) advisory.